backend/endpoints/rooms.py

```diff
@@ -127,7 +127,7 @@ async def remove_member(
             detail="user not a member of the room",
         )
 
-    if admin_id not in room_data["room_members"]:
+    if admin_id is not None and admin_id not in room_data["room_members"]:
         raise HTTPException(
             status_code=status.HTTP_404_NOT_FOUND,
             detail="admin id specified not a member of the room",
```

Comment:
Okay, I just tested the endpoint. I couldn't remove a regular member from a channel with or without an admin_id. this is what's returned.
{
  "detail": "\"unable to remove room member\""
}